src/jvm/test/resources/expected_logs/run_concurrent_test/main_thread_blocked.txt

```diff
@@ -37,24 +30,12 @@ Detailed trace:
 |           Unsafe#1.putReferenceOpaque(Thread#1, 76, ConditionObject#1) at LockSupport.setCurrentBlocker(LockSupport.java:161)                                |
-|         canReacquire(ConditionNode#1): false at AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1614)                       |
-|           node.prev ➜ null at AbstractQueuedSynchronizer$ConditionObject.canReacquire(AbstractQueuedSynchronizer.java:1527)                                  |
-|         node.status ➜ 3 at AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1618)                                            |
 |         ForkJoinPool.managedBlock(ConditionNode#1) at AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)                 |
-|           ForkJoinPool.unmanagedBlock(ConditionNode#1) at ForkJoinPool.managedBlock(ForkJoinPool.java:3436)                                                  |
-|             blocker.isReleasable(): false at ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3465)                                                             |
-|               status ➜ 3 at AbstractQueuedSynchronizer$ConditionNode.isReleasable(AbstractQueuedSynchronizer.java:502)                                       |
-|             blocker.block() at ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3465)                                                                           |
-|               /* The following events repeat infinitely: */                                                                                                  |
-|           ┌╶> isReleasable(): false at AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)                                   |
-|           |     status ➜ 3 at AbstractQueuedSynchronizer$ConditionNode.isReleasable(AbstractQueuedSynchronizer.java:502)                                     |
-|           |   LockSupport.park() at AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)                                      |
-|           |     PARK at LockSupport.park(LockSupport.java:341)                                                                                               |
-|           └╶╶ switch (reason: active lock detected)                                                                                                          |
+|           /* The following events repeat infinitely: */                                                                                                      |
+|       ┌╶> LockSupport.park() at AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)                                          |
+|       |     PARK at LockSupport.park(LockSupport.java:341)                                                                                                   |
```

Comment:
PARK at LockSupport.park(LockSupport.java:341) -- this event should not be listed in the execution trace